alifedata_phyloinformatics_convert/_aux/_alifestd_validate.py

```diff
@@ -0,0 +1,93 @@
+"""This file has been copied from hstrat (https://github.com/mmore500/hstrat/blob/master/hstrat/_auxiliary_lib/_alifestd_validate.py).
```

Comment:
There's a new subfolder called _impl, I'd move _aux itemsf into there.fte

Reply:
Done in 7f3fbcc